src/routes.jsx

```diff
@@ -98,7 +98,7 @@ export const makeRoutes = (routing) => {
                 <Route path="/loading/comparison" onEnter={()=>{$(document).scrollTop(0)}} component={GroupComparisonLoading} />
                 <Route path="/results/legacy_submission" onEnter={handleLegacySubmission} component={getBlankPage()} />
 
-                <Route path="/results(/:tab)" onEnter={()=>{$(document).scrollTop(0)}} getComponent={lazyLoadComponent(ResultsViewPage)} />
+                <Route path="/results(/:tab)" onEnter={()=>{}} getComponent={lazyLoadComponent(ResultsViewPage)} />
```

Comment:
why?

Reply:
because when user switches tabs on results view, Niki wants browser to stay where it is, not go to top.

Reply:
ah okay